app/views/devise/confirmations/new.html.slim

```diff
@@ -1,3 +1,4 @@
+= stylesheet_link_tag 'store'
```

Comment:
Devise tiene una forma de configurar su propio layout mira este tutorial https://github.com/heartcombo/devise/wiki/How-To:-Create-custom-layouts